arkouda/numeric.py

```diff
@@ -422,16 +428,26 @@ def cos(pda: pdarray) -> pdarray:
     return create_pdarray(repMsg)
 
 
-@typechecked
+def _hash_helper(a):
```

Comment:
should this  be `a: hashable`?

Reply:
sure! It's not user facing or typechecked so idk if it's completely necessary, but i don't mind making the change it could be helpful for devs who want to know what types this functions expects